include/small/ibuf.h

```diff
@@ -33,6 +33,15 @@
 #include <stddef.h>
 #include <assert.h>
 
+#include "small_config.h"
+
+#ifdef ENABLE_ASAN
+#  include <sanitizer/asan_interface.h>
+#else
+#  define ASAN_POISON_MEMORY_REGION(addr, size) ((void)(addr), (void)(size))
+#  define ASAN_UNPOISON_MEMORY_REGION(addr, size) ((void)(addr), (void)(size))
+#endif
```

Comment:
Should be in `util.h` because it may be used for other allocators.

Reply:
Done.

<details>
<summary>diff</summary>

```diff
diff --git a/include/small/ibuf.h b/include/small/ibuf.h
index 04c1568..1e9c848 100644
--- a/include/small/ibuf.h
+++ b/include/small/ibuf.h
@@ -33,14 +33,7 @@
 #include <stddef.h>
 #include <assert.h>
 
-#include "small_config.h"
-
-#ifdef ENABLE_ASAN
-#  include <sanitizer/asan_interface.h>
-#else
-#  define ASAN_POISON_MEMORY_REGION(addr, size) ((void)(addr), (void)(size))
-#  define ASAN_UNPOISON_MEMORY_REGION(addr, size) ((void)(addr), (void)(size))
-#endif
+#include "util.h"
 
 #if defined(__cplusplus)
 extern "C" {
diff --git a/include/small/util.h b/include/small/util.h
index 87a2489..845c7b1 100644
--- a/include/small/util.h
+++ b/include/small/util.h
@@ -250,4 +250,9 @@ small_asan_header_from_payload(void *payload)
 void
 small_asan_free(void *header);
 
-#endif /* ifdef ENABLE_ASAN */
+#else /* ifndef ENABLE_ASAN */
+
+#define ASAN_POISON_MEMORY_REGION(addr, size) ((void)(addr), (void)(size))
+#define ASAN_UNPOISON_MEMORY_REGION(addr, size) ((void)(addr), (void)(size))
+
+#endif /* ifndef ENABLE_ASAN */

```

</details>